frontend/src/component/pages/admin/changelog/ManageChangeLogs.scss

```diff
@@ -0,0 +1,41 @@
+@import 'src/component/styles/color';
+@import 'src/component/styles/layouts';
+@import 'src/component/styles/shape';
+
+.manage-change-logs {
+  .create-change-button {
```

Comment:
It seems that the button is rounded in the design. Let's add rounded style in button component to support that!